modules/gapi/include/opencv2/gapi/infer/onnx.hpp

```diff
@@ -115,13 +117,38 @@ template<typename Net> class Params {
         return *this;
     }
 
-    Params<Net>& cfgPostProc(const std::vector<cv::GMatDesc> &outs,
+    Params<Net>& cfgPostProc(const std::vector<cv::GMatDesc> &out_metas,
                              const PostProc &pp) {
-        desc.out_metas = outs;
+        desc.out_metas        = out_metas;
         desc.custom_post_proc = pp;
         return *this;
     }
 
+    Params<Net>& cfgPostProc(std::vector<cv::GMatDesc> &&out_metas,
+                             PostProc &&pp) {
+        desc.out_metas        = std::move(out_metas);
+        desc.custom_post_proc = std::move(pp);
+        return *this;
+    }
```

Comment:
I believe doxygen comments are really missing here since the semantics of these methods is not very trivial

Reply:
Added.